alex_questions.md

```diff
@@ -0,0 +1,19 @@
+# direct TX
+
+- What format can we expect for the direct TX on the DA? some wrapper type would be useful to unpack and distinct from
+  random bytes
+- Should we always include direct TX although they may be duplicates to mempool TX? spike: yes, first one wins.
```

Comment:
yes, first one wins. even if there's a duplicate, the duplicate would just fail automatically due to usual tx replay protections in state machines like nonces